content/v2.0/cloud/account-management/offboarding.md

```diff
@@ -58,6 +56,8 @@ To request a backup of data in your {{< cloud-name "short" >}} instance, contact
 
     {{< nav-icon "usage" >}}
 
-2. Click **Cancel Service**.
-3. Select **I understand and agree to these conditions**, and then click **I understand, Cancel Service.**
-4. Click **Confirm and Cancel Service**. Your payment method is charged your final balance immediately upon cancellation of service.
+2. Do one of the following:
+
+  - If you subscribed to an InfluxDB Cloud 2.0 plan through **AWS Marketplace**, click the **AWS** link, and then click **Unsubscribe**.
+  
+  - If you subscribed to an InfluxDB Cloud 2.0 plan through **InfluxData**, click **Cancel Service**. Select **I understand and agree to these conditions**, and then click **I understand, Cancel Service.** Click **Confirm and Cancel Service**. Your payment method is charged your final balance immediately upon cancellation of service.
```

Comment:
- @bpalani - did we clean up/minimize this cancel text (I vaguely recall a PR)?

- Will Free Plan folks be looking to cancel their "service" being they're not paying for a subscription? Should we specify "Usage-Based Plan" here? Other thoughts?

Reply:
We don't provide a flow for free users to cancel their accounts -- there's really nothing to cancel. I think it's ok to specify `Usage-Based Plan`

If they want to _remove_ their account for GDPR reasons, that's a separate manual process requiring legal. Don't think we should document that one.